server/src/models/v2/in/CreateAppModel.js

```diff
@@ -0,0 +1,38 @@
+const Joi = require('@hapi/joi')
+const { isSemver } = require('../../helpers')
+
+const { AppTypes } = require('../../../enums')
+
+const CreateModelAppData = Joi.object().keys({
+    name: Joi.string(),
+    description: Joi.string(),
+    appType: Joi.string().valid(...AppTypes),
+    sourceUrl: Joi.string().uri(),
+    developer: Joi.object().keys({
```

Comment:
I think this model will change a bit. Depending on what we decide for https://github.com/dhis2/notes/discussions/260 , developer-information will probably change. Do we merge this before then, or decide on an API for before?

I think we should remove the `developer`-object and keep `developerEmail` and `organisationId` outside of the object.